addon-test-support/helpers.js

```diff
@@ -189,10 +189,10 @@ export function find(selector, contextEl) {
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
     selected = document.querySelectorAll(`${settings.rootElement} ${selector}`);
   }
-  if (!settings.useJQueryWrapper) {
-    return (selected.length === 1) ? selected[0] : selected;
+  if (settings.useJQueryWrapper) {
```

Comment:
Yesterday I missed the conversation around `find` and configuration, but the idea is that if there is only one matching node, it returns it and if there is many you return a `NodeList`?

I'd be cool with returning native DOM elements and letting users do `$(find('selector'))` if they want jquery.